ipfs/ipfs.go

```diff
@@ -0,0 +1,72 @@
+package ipfs
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+)
+
+const (
+	addAPI = "api/v0/add"
+)
+
+// IPFS is the struct that contains all informations to access an IPFS node
+type IPFS struct {
+	client   *http.Client
+	endpoint string
+}
+
+// Response return the response of a file uploaded
+type Response struct {
+	Name string
+	Hash string
+	Size string
+}
+
+// New creates a new IPFS client method
+func New() *IPFS {
+	return &IPFS{
+		client:   &http.Client{},
+		endpoint: "https://ipfs.infura.io:5001/",
```

Comment:
it's be good to move this to _config_ pkg

Reply:
This package should not be related to MESG, introducing the config here will strongly coupled this lib. Also this is a package only used in the CLI (that should not access to the config, if it needs to it should contact the Info api https://github.com/mesg-foundation/core/pull/748)

Reply:
then it makes sense to move it under _commands_ pkg?

Reply:
no this is an independent package, it shouldn't be linked to the commands, i would be agree maybe to have it int the `x` packages but inside command no, the goal is to have it independent from the rest.